docs/modules/ROOT/pages/how-tos/configuring/customizing-the-build.adoc

```diff
@@ -1 +1,168 @@
-= Customizing the build pipeline
\ No newline at end of file
+= Customizing the build pipeline
+
+To meet your specific needs, you can customize the way that {ProductName} builds components. You can customize builds in two ways:
+
+* Change the parameters of the build pipeline of a component.
+* Extend the pipeline with your own tasks (or delete existing tasks).
+
+For example, you can change for how long {ProductName} saves the container images of component, by changing the value of the `image-expires-after` parameter. Or you can ensure that the container images it builds for a component are compliant with regulations for your industry, by adding a compliance check as a task to its build pipeline. 
+
+
+.Procedure
+
+To customize the build pipeline in any way, start by taking the following steps:
+
+. In your preferred IDE, navigate to the `.tekton` directory in the repository of your component.
+
+. Open the `testrepo-pull-request.yaml` file in the repository.
+ 
+
+== Changing parameters
+
+.Procedure
+
+. In this `.yaml` file in the `.tekton` directory, customize the parameters of the build pipeline by changing the pipeline `params`:
+.. Pipeline params have the paths `.spec.params` and `.pipelineSpec.params`
+.. Change the default `value` of existing parameters. For example, you can set the value of `image-expires-after` to `2w`, instead of `5d` 
+.. Add parameters. New parameters must include a `name` and `value`.
+. Commit your changes to the repository of the component.
+
+
+== Extending the build pipeline with your own tasks
+
+Before you extend your build pipeline, be aware that doing so causes builds to fail an link:https://enterprisecontract.dev/docs/ec/main/index.html[Enterprise Contract] check, if you are using the EC. To resolve this issue, please reference the next procedure after completing this one.  
+
+.Procedure
+
+. In each `.yaml` file in the `.tekton` directory, add a new task under `tasks`. 
```

Comment:
Maybe we can say under `.spec.pipelineSpec.tasks` to be more explicit since the term "tasks" shows up in multiple locations?